resources/sup.rb

```diff
@@ -0,0 +1,67 @@
+#
+# Copyright:: Copyright (c) 2017 Chef Software, Inc.
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+resource_name :hab_sup
+provides :hab_sup
+
+property :permanent_peer, [TrueClass, FalseClass]
+property :listen_gossip, String
+property :listen_http, String
+property :override_name, String, default: 'default'
+property :org, String, default: 'default'
+property :peer, String
+property :ring, String
+
+action_class do
```

Comment:
while not yet a rule at some point we're going to tell people to shove these at the bottom for compatibility with older chef clients. You might as well do it now even though you don't support those. It'll avoid the failure in FC latger